packages/frontend/src/features/sqlRunner/components/SqlEditor.tsx

```diff
@@ -115,7 +115,8 @@ const LIGHTDASH_THEME = {
     colors: {
         'editor.background': '#FFFFFF',
         'editor.foreground': '#333333',
-        'editor.lineHighlightBackground': '#ffffff',
+        'editor.lineHighlightBackground': '#f8f8f8',
```

Comment:
in the previous sql runner, the line number was also highlighted, is this possible here ? if not, this should be enough..

Reply:
I'm not too sure what keyword on the theming I need to add, but this background should be enough for now 👍